Dockerfile.template

```diff
@@ -3,6 +3,8 @@ FROM balenalib/%%RESIN_MACHINE_NAME%%-debian
 RUN apt-get update && apt-get install -y curl wget build-essential libelf-dev awscli
 COPY . /usr/src/app
 WORKDIR /usr/src/app
-RUN ./build.sh %%RESIN_MACHINE_NAME%% '2.26.0+rev1.prod' example_module
+
+ENV VERSION '2.29.0+rev1.prod'
```

Comment:
I'm thinking maybe we should stop providing a version here and instruct the user to fill it in.

Reply:
@agherzan yeah, though as an example it might be a good idea to make it buildable. People will have to modify this for their projects, not just build this repo, so might as well have this repo stand-alone, so they can build it. Anyways, there are more things indeed to improve in general, so making this non-hardcoded, while still user friendly is a very good point.